Lombiq.HelpfulLibraries/Libraries/Utilities/NumberExtensions.cs

```diff
@@ -58,5 +58,11 @@ public static class NumberExtensions
         /// </summary>
         public static int ToTechnicalInt(this string number) =>
             int.TryParse(number, NumberStyles.Any, CultureInfo.InvariantCulture, out var result) ? result : -1;
+
+        /// <summary>
+        /// Returns the number of digits in the given <see cref="int"/>.
+        /// </summary>
+        /// <param name="number">The number to count the digits of.</param>
+        public static int DigitCount(this int number) => (int)Math.Log10(number) + 1;

```

Comment:
Don't you want to make another extension for the D thing too to make the whole thing less technical? Something like `int PadZeroes(this number, int length) => number.ToString("D" + length.ToTechnicalString());`

Reply:
I didn't want to but now I did!

Reply:
![image](https://user-images.githubusercontent.com/4007293/152427485-014fb329-a85d-483a-994c-a7d622d28083.png)

Reply:
It's a pleasure to follow you two! 😁

Reply:
:D